boba-bindings/bindings/registry.go

```diff
@@ -0,0 +1,39 @@
+package bindings
```

Comment:
Having spent a bit more time here lately, I'm starting to understand what's going on a bit better.

As best as I can tell, the `_more.go` files bind into these variables like `layouts` files, and are generated.

It's a bit weird to me, that this file is not generated, while the rest of the files in this binding directory are.  But, I guess this is not our pattern.

Still, it might be helpful to describe exactly what's going on here in the README

Reply:
Added